pages/01.guides/03.core/05.ContinuationTokens/doc.md

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Continuation Tokens'
+---
+
+# Continuation Tokens
+
+Continuation tokens are a method used for Pagination in endpoints starting with V2.
+
+When you make a request to an endpoint that supports continuation tokens you will get the response and a continuation token back. This is usually found in the headers and often(but not always) included in a link header.
```

Comment:
its only included in a link header for chat v2 api, should we change that?

Reply:
Im hesitant. I've seen it in both headers all by itself and the link headers so I wanted to convey that its included in both.

Reply:
For sure, let me know once we come to conclusion so that I can be consistent with other apis :)

Reply:
imo having the link header alone is good. It removes the onus on clients to form URLs for further pages, and removes the responsibility from miscellaneous service owners to all make sure their parameters match: as long as they all return a link header that works, that's all they need to do. And if we want to include more info or use a data store that uses pages or whatever (e.g. if money-related stuff remains backend by a SQL store which uses limits of offsets) then that can all be done without any client changes.

Reply:
For now I've seen in it both. So leaving this here. we can update if we reach a quorum with the team.

I agree the link headers are powerful :)